subsys/bluetooth/Kconfig

```diff
@@ -117,6 +117,7 @@ config BT_CONN_TX
 
 config BT_LE_MAX_LOCAL_SUPPORTED_FEATURE_PAGE
 	int "Maximum supported feature page"
+	default 1 if BT_FRAME_SPACE_UPDATE
```

Comment:
Would something break if the application sets this config to 0 while `BT_FRAME_SPACE_UPDATE` is enabled? If so, should this config be promptless or update the `range` as well if `BT_FRAME_SPACE_UPDATE`?

Reply:
That's a good point, thanks